pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -415,3 +419,16 @@ def teardown(self) -> None:
             self.lightning_module.cpu()
             # clean up memory
             torch.cuda.empty_cache()
+
+    @staticmethod
+    def _clean_logger(trainer: "pl.Trainer") -> None:
+        loggers = trainer.logger._logger_iterable if isinstance(trainer.logger, LoggerCollection) else [trainer.logger]
+        for logger in loggers:
+            if isinstance(logger, TensorBoardLogger) and logger._experiment is not None:
+                rank_zero_warn(
```

Comment:
@tchaton Is it necessary to have this warning? This info won't be necessary for majority of users. 

And this warning pops up every time a user trains with `ddp_spawn`, `ddp_spawn_sharded` and `tpu_spawn` as TensorBoard logger is the default.